### PR TITLE
Created test file for next steps

### LINKED ISSUE #1548: Create test file for Next Steps

### PR DESCRIPTION
**Pull request by SupeDeDupe**
Closes #1548

### CODE REVIEW

**Comment by smcmurtry** [comment]
Can you add some more tests? Ideally the tests should cover the main functionality for the component. For next steps that is currently just rendering static content. So you could test:
1. that the bullet list is rendered
2. that the 2 cards are rendered

File: __tests__/components/next_steps_test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  beforeEach(() => {
+    props = {
+      t: translate,
+      mapUrl: { query: {}, route: "/map" }
+    };
+
+    mockStore = configureStore();
+    reduxState = {};
+    props.store = mockStore(reduxState);
+    props.reduxState = reduxState;
```
Comment (smcmurtry) [inline __tests__/components/next_steps_test.js:19]: I think you can delete lines 19 and 21 and this should still work

**Comment by SupeDeDupe** [comment]
Not sure how to fix this warning:
screen shot 2018-12-11 at 10 32 08 am

File: __tests__/components/next_steps_test.js (hunk 1 above)
Comment (SupeDeDupe) [reply __tests__/components/next_steps_test.js:19]: I think I removed them

File: __tests__/components/next_steps_test.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const { axe, toHaveNoViolations } = require("jest-axe");
+expect.extend(toHaveNoViolations);
+
+describe("NextSteps", () => {
+  let props;
+  let reduxState;
+  beforeEach(() => {
+    props = {
+      t: translate,
+      mapUrl: { query: {}, route: "/map" }
```
Comment (smcmurtry) [inline __tests__/components/next_steps_test.js]: This should actually just be a string - it's different from the url prop that is passed to each page, it's the result of the `getMapUrl` function in selectors/urls.js. That should fix your warning.

**Review by smcmurtry — APPROVED** [review]
Good work! 🦑

File: __tests__/components/next_steps_test.js (hunk 2 above)
Comment (SupeDeDupe) [reply __tests__/components/next_steps_test.js]: But how do I make it a string and still the thing I want?

Comment (smcmurtry) [reply __tests__/components/next_steps_test.js]: `mapUrl: "/map"`

Comment (SupeDeDupe) [reply __tests__/components/next_steps_test.js]: Ok but then what do I put in the test for it on line 69? I think if I write "/map" then does the test still make sense??

**Comment by smcmurtry** [comment]
[Users are given options for what the next step is in their current stage](https://trello.com/c/rHNRWLYF/2-users-are-given-options-for-what-the-next-step-is-in-their-current-stage)

**Pull request merged**